releases/net471/KnownIssues/521954 - BCL CultureAwareComparer with ignore casing on serialized on previous versions of .NET do not correctly deserialize on .NET 4.7.1.md

```diff
@@ -21,8 +21,12 @@ P:System.StringComparer.CurrentCultureIgnoreCase
 
 Ensure that both serialization and deserialization occurs on systems running a version of the .NET Framework starting with 4.7.1.
 
+## Resolution
+
 If you are running on Windows 10 Fall Creators Update, an update is available that addresses this issue. Select one of the following links based on your processor architecture:
 
-x86: https://go.microsoft.com/fwlink/?linkid=862758
+    x86: https://go.microsoft.com/fwlink/?linkid=862758
```

Comment:
Are the x86 and x64 links superseded by the fix, @vivmishra?

Reply:
Those are for Fall Creators release that is not covered by the down-level ZDP. 
So, we still need those.